File: Kliens-RAPC9Y-Backend/Kliens-RAPC9Y-Backend/Models/Game.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using System.Text.Json.Serialization;

namespace Kliens_RAPC9Y_Backend.Models
{
    public class Game
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }

        [Required]
        public string GameName { get; set; }

        [Required]
        public int ReleaseDate { get; set; }

        [Required]
        public string GameCover { get; set; }

        [NotMapped]
        public virtual ICollection<Boss> Bosses { get; set; }

        public Game()
        {
            this.Bosses = new HashSet<Boss>();
        }

    }
}

File: KFrontend/Frontend/src/app/_models/game.ts
import { Boss } from "./boss"

export class Game {
  public id : number = 0
  public gameName : string = ""
  public releaseDate : number = 0
  public gameCover : string = ""
  public bosses : Array<Boss> = []
  public gameDescription : string = ""

  public createBosses(bossList : Array<Boss>){
    
  }
}